providers/infrastructure-vsphere/v1.5.1/cconly/base.yaml

```diff
@@ -585,6 +585,58 @@ spec:
         default:
           activate: true
           daysBefore: 90
+  - name: gpu
```

Comment:
these variables need to be added in the mappings here I think?
 https://github.com/vmware-tanzu/tanzu-framework/blob/822456a9d4f1f131b7193c7e937f151522f81881/tkg/constants/clusterclass_mapping.go#L202

Reply:
What's the idea behind adding the variables to this place? And complex types like maps or arrays, how are those being handled?

Reply:
@yastij